app/main.py

```diff
@@ -1,8 +1,45 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self, distance_from_city_center: float, clean_power: int,
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability